vuepress/docs/next/docs/reference/deployment-structure.md

```diff
@@ -42,7 +42,7 @@ The init containers automatically restore a backup included in your application
 The **entando-de-app** is a J2EE application and an instance of the [entando-core](#entando-core). It provides pathways for Entando Components and the server image required by the Entando Operator to manage the deployment. The pom.xml for the application reveals its dependencies.
 
 * GitHub: <https://github.com/entando/entando-de-app/>
-* DockerHub: <https://hub.docker.com/repository/docker/entando/entando-de-app-eap>,<https://hub.docker.com/repository/docker/entando/entando-de-app-wildfly>
+* DockerHub: 
```

Comment:
@pietrangelo need the new DockerHub addr for the 7.2 de-app

Reply:
https://registry.hub.docker.com/r/entando/entando-de-app-eap, https://registry.hub.docker.com/r/entando/entando-de-app-wildfly, https://registry.hub.docker.com/r/entando/entando-de-app-tomcat